accelerator-home-ui/src/api/TTSApi.js

```diff
@@ -0,0 +1,184 @@
+/**
+ * If not stated otherwise in this file or this component's LICENSE
+ * file the following copyright and licenses apply:
+ *
+ * Copyright 2020 RDK Management
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ **/
+import ThunderJS from 'ThunderJS';
+import { CONFIG } from '../Config/Config'
+import { Metrics } from '@firebolt-js/sdk';
+
+const thunder = ThunderJS(CONFIG.thunderConfig)
+const callsign = 'org.rdk.TextToSpeech'
+const errorName = 'TextToSpeechError'
+
+export default class TTSApi {
+    activate() {
+        return new Promise((resolve, reject) => {
+            thunder.Controller.activate({ callsign: callsign })
+                .then(() => {
+                    resolve(true)
+                })
+                .catch(err => {
+                    console.log('Error Activation', err)
```

Comment:
Please use appropriate log level; in this case apt one is log.error() since its from a catch block.
Please address similarly in this complete change